docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -144,34 +144,45 @@ all labeled and additional information will be included below.
 
 
 .. list-table:: Outputs from Pregnancy Decision Tree
-  :widths: 3 15 15
+  :widths: 3 15 15 15
   :header-rows: 1
 
   * - Input
     - Data Source 
-    - Notes
+    - Overall stratification
+    - Use case 
   * - ANC attendance
     - Decision tree point
-    - 
+    - Age, scenario 
+    - Decision tree value & export value 
   * - Ultrasound status (AI assisted, standard, none)
     - Decision tree point
-    - 
+    - Scenario
+    - Decision tree value & export value 
   * - Gestational age at birth
     - :ref:`Pregnancy model <other_models_pregnancy_closed_cohort_mncnh>`
-    - 
+    - Decision tree value & export value 
+    - Scenario
   * - Gestational age stated
     - Decision tree value
-    - We should track both the real GA at birth and the believed GA 
-  * - Birthweight
-    - :ref:`Pregnancy model <other_models_pregnancy_closed_cohort_mncnh>`
     - 
-  * - Low birthweight status
+    - Decision tree value
+  * - Birthweight
     - :ref:`Pregnancy model <other_models_pregnancy_closed_cohort_mncnh>`
     - 
```

Comment:
Similar to above, I think birthweight (LBWSG) is scenario and age specific and should be checked in a model. If you were thinking intrapartum or if I'm wrong about age specific (see above) then ignore.